### PR TITLE
log-backup: eliminate some verbose logs

### LINKED ISSUE #14453: log-backup: verbose log printed even there isn't a log bakcup task
### What version of TiKV are you using?
Current master.

### What operating system and CPU are you using?
Irrelative.

### Steps to reproduce
Start a TiKV server.

### What did you expect?
There shouldn't be too many logs about log backup because there isn't log backup task.

### What did happened?
```log
[2023/03/23 09:47:24.397 +00:00] [INFO] [[subscription_manager.rs:480](http://subscription_manager.rs:480/)] ["getting checkpoint"] [defined_by_region=None]
[2023/03/23 09:47:34.398 +00:00] [INFO] [[subscription_manager.rs:410](http://subscription_manager.rs:410/)] ["backup stream: on_modify_observe"] [op="ResolveRegions { min_ts: TimeStamp(440287849189736449), callback: fn { .. } }"]
[2023/03/23 09:47:34.398 +00:00] [INFO] [[subscription_manager.rs:480](http://subscription_manager.rs:480/)] ["getting checkpoint"] [defined_by_region=None]
[2023/03/23 09:47:44.399 +00:00] [INFO] [[subscription_manager.rs:410](http://subscription_manager.rs:410/)] ["backup stream: on_modify_observe"] [op="ResolveRegions { min_ts: TimeStamp(440287851811438593), callback: fn { .. } }"]
[2023/03/23 09:47:44.399 +00:00] [INFO] [[subscription_manager.rs:480](http://subscription_manager.rs:480/)] ["getting checkpoint"] [defined_by_region=None]
```

### PR DESCRIPTION
**Pull request by YuJuncen**
### What is changed and how it works?

Issue Number: Close #14453 

What's Changed:
Remove handler for `PrepareMinTsForResolve` when there isn't any task. 

```commit-message
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
1. Start a new cluster in master. -- Many logs described in #14453 printed.
2. Start a new cluster in this PR. -- Logs described in #14453 don't print again as long as no log backup task triggered.

### Release note 

```release-note
Fixed a bug that may cause TiKV print too many meanless logs about log backup.
```

### CODE REVIEW

File: components/backup-stream/src/metrics.rs [hunk 1]
```diff
@@ -155,9 +156,11 @@ lazy_static! {
         &["stage"]
     )
     .unwrap();
-    pub static ref LOST_LEADER_REGION: IntCounter = register_int_counter!(
```
Comment (YuJuncen) [inline components/backup-stream/src/metrics.rs:158]: For reviewers: this series hasn't been uploaded into any of the official Grafana dashboards, it is probably safe to rename it.

**Review by 3pointer — APPROVED** [review]

File: components/backup-stream/src/metrics.rs [hunk 2]
```diff
@@ -167,3 +170,17 @@ lazy_static! {
     )
     .unwrap();
 }
+
+make_static_metric! {
+    pub label_enum MIscEventsName {
```
Comment (Leavrth) [inline components/backup-stream/src/metrics.rs:175]:
```suggestion
    pub label_enum MiscEventsName {
```

**Review by Leavrth — APPROVED** [review]

**Review by hicqu — APPROVED** [review]

**Pull request merged**